dip-template/runtimes/dip-consumer/src/xcm_config.rs

```diff
@@ -74,7 +75,7 @@ impl xcm_executor::Config for XcmConfig {
 	type IsTeleporter = ();
 	type MaxAssetsIntoHolding = ConstU32<64>;
 	type MessageExporter = ();
-	type OriginConverter = SiblingParachainAsNative<cumulus_pallet_xcm::Origin, RuntimeOrigin>;
+	type OriginConverter = AccountIdJunctionAsParachain<ConstU32<2_000>, cumulus_pallet_xcm::Origin, RuntimeOrigin>;
```

Comment:
```
type OriginConverter  = SovereignSignedViaLocation<([ForeignChainAliasAccount<AccountId>](https://github.com/paritytech/polkadot/blob/d67d47f4bc330988dad0a97f5a537a4413ee90e7/xcm/xcm-builder/src/location_conversion.rs#L71)), RuntimeOrigin>;
```
What are the cons of using this type of configuration?
Can the consumer filter out any calls and only allow the call function to be executed via XCM?

Reply:
This is a good suggestion. I think I will include it once the PR is rebased on top of 0.9.42.